frontend/src/metabase/admin/settings/components/SettingsEmailForm.jsx

```diff
@@ -252,40 +257,42 @@ export default class SettingsEmailForm extends Component {
       saveButtonText = saveSettingsButtonStates[submitting];
 
     return (
-      <form noValidate>
-        <ul>
-          {settings}
-          <li className="m2 mb4">
-            <button
-              className={cx(
-                "Button mr2",
-                { "Button--primary": !disabled },
-                { "Button--success-new": submitting === "success" },
-              )}
+      <ul>
+        {settings}
+        <li className="m2 mb4">
+          <Button
+            primary={!disabled}
+            className={cx({ "Button--success-new": submitting === "success" })}
+            disabled={disabled}
+            onClick={this.updateEmailSettings.bind(this)}
+          >
+            {saveButtonText}
+          </Button>
+          {valid && !dirty && submitting === "default" ? (
+            <Button
+              className={cx("ml1", {
+                "Button--success-new": sendingEmail === "success",
+              })}
               disabled={disabled}
-              onClick={this.updateEmailSettings.bind(this)}
+              onClick={this.sendTestEmail.bind(this)}
             >
-              {saveButtonText}
-            </button>
-            {valid && !dirty && submitting === "default" ? (
-              <button
-                className={cx("Button", {
-                  "Button--success-new": sendingEmail === "success",
-                })}
-                disabled={disabled}
-                onClick={this.sendTestEmail.bind(this)}
-              >
-                {emailButtonText}
-              </button>
-            ) : null}
-            {formErrors && formErrors.message ? (
-              <span className="pl2 text-error text-bold">
-                {formErrors.message}
-              </span>
-            ) : null}
-          </li>
-        </ul>
-      </form>
+              {emailButtonText}
+            </Button>
+          ) : null}
+          <Button
```

Comment:
Should we hide or disable the button if the form is already cleared?

Reply:
It looks like `disabled` is functioning correctly so the button is never clickable except when the form has stuff to clear. Agree it would probably make sense to only show the form if it's got values in it already.